proxy/bootstrap/src/main/java/org/apache/shardingsphere/proxy/arguments/BootstrapArguments.java

```diff
@@ -97,4 +118,13 @@ private String paddingWithSlash(final String pathArg) {
         }
         return result.toString();
     }
+    
+    private boolean isValidPath(final String path) {
+        try {
+            Paths.get(path);
+        } catch (InvalidPathException | NullPointerException ex) {
+            return false;
```

Comment:
My understanding is that if user specified an invalid path, the socket will not be present and no error will occur. Am I right?

Reply:
yes, it was designed that way.

Reply:
of course, we can throw a error in this.

Reply:
I prefer throwing an exception if path was invalid.